doc/develop/microscope.rst

```diff
@@ -91,6 +91,38 @@ Detectors:
 System:
  * lens: Contains parameters concerning the parabolic mirror and the lens system.
 
+Roles of components found in the CRYO/ENZEL system:
+-----------------------------------------------------
+Emitters:
+ * e-beam: Electron beam of the SEM to scan the sample.
+ * light: Controls the excitation light of the FM.
+
+Actuator:
+ * ebeam-focus: Changes the focus position of the e-beam. It has one axis: z. 
+ * filter: Emission filter to select a specific wavelength band.
+ * stage: Moves the sample. It has 3 linear axes (x, y, z) and 2 rotational axes (rx, rz). stage component has the following metadata:
+    1. FAV_POS_DEACTIVE.
+    2. FAV_POS_ACTIVE.
+    3. FAV_POS_COATING.
+    4. FAV_POS_AREA.
+    5. ION_BEAM_TO_SAMPLE_ANGLE.
```

Comment:
Would be nice to add a short description of what these metadata set. It should be found in the yaml file.

Reply:
Okay. But not all of them are provided in the yaml file

Reply:
You can find generic definitions in:
src/odemis/model/_metadata.py